sources/zafira-models/src/main/java/com/qaprosoft/zafira/models/db/Invitation.java

```diff
@@ -25,6 +25,7 @@
     private Status status;
     private User.Source source;
     private Long groupId;
+    private String invitationUrl;
```

Comment:
I say this one should be called simply `url` since it's pretty clear that it is actually an invitation url because of the fact that it resides in a class called `Invitation` :)